build_projects/dotnet-cli-build/AddMetadataIsPE.cs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System;
+using System.IO;
+using System.Reflection.PortableExecutable;
+using System.Collections.Generic;
+
+namespace Microsoft.DotNet.Cli.Build
+{
+    public class AddMetadataIsPE : Task
```

Comment:
I was trying to develop a reusable pattern as I didn't find one on the Web. Add*Metadata and AddMetadata* both make sense to me. AddMetadata* felt like it would group together more nicely in intellisense, etc. and would not leave room for ambiguity with other Tasks.